manager/Kconfig

```diff
@@ -0,0 +1,11 @@
+menu "Smartknob-HA Components"
+
+    menu "Manager"
+
+        config NUM_COMPONENTS
+            int "Number of components in use from the SDK"
+            default 1
+            help
+                You should probably not touch this unless you've added a component
+    endmenu
+endmenu
```

Comment:
There is a Kconfig entry for `NUM_COMPONENTS`, but it is not used in any of the sourcefiles and is replaced with a define of the same name.